src/mmw/js/src/modeling/models.js

```diff
@@ -508,15 +508,20 @@ function getControlsForModelPackage(modelPackageName, options) {
 
 function getHumanReadableLabel(value) {
     var mapping = {
+        'chaparral': 'Chaparral',
         'commercial': 'Commercial',
+        'desert': 'Desert',
         'forest': 'Forest',
         'grassland': 'Grassland',
         'hir': 'HIR',
         'lir': 'LIR',
         'pasture': 'Pasture',
         'row_crop': 'Row Crop',
+        'sg_prairie': 'Short Grass Prairie',
+        'tg_prairie': 'Tall Grass Prairie',
         'turf_grass': 'Turf Grass',
         'wetland': 'Wetland',
+
```

Comment:
Here, I was trying to separate the land uses from the BMPs, as is done in patterns.js.  I did have an extra newline in patterns.js, though (two instead of one).